Modules/IO/NIFTI/test/itkNiftiImageIOTest5.cxx

```diff
@@ -132,6 +132,96 @@ SlopeInterceptTest()
   return maxerror > 0.00001 ? EXIT_FAILURE : EXIT_SUCCESS;
 }
 
+template <typename PixelType>
+int
+SlopeInterceptWriteTest()
+{
+  //
+  // fill out an image and write it as nifti.
+  const char *filename = "SlopeIntercept.nii";
+  typedef typename itk::Image<PixelType,3> OutputImageType;
+  OutputImageType::RegionType region;
```

Comment:
Since `OuputImageType::RegionType` is a [dependent type](https://en.cppreference.com/w/cpp/language/dependent_name) this needs the `typename` keyword.